src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+function toNumber(salary) {
+  return parseFloat(salary.substring(1).replaceAll(',', ''));
+}
+
+function sortList(list) {
+  const lst = document.querySelector(list);
```

Comment:
```suggestion
  const list = document.querySelector(list);
```
don't use short forms

Reply:
list is already a function argument, I can't use it